apps/myaccount/src/extensions/configs/scim.ts

```diff
@@ -35,11 +35,11 @@ export const SCIMConfigs: SCIMConfigInterface = {
     },
 
     scimEnterpriseUserClaimUri: {
-        accountDisabled: "urn:scim:wso2:schema.accountDisabled",
-        accountLocked: "urn:scim:wso2:schema.accountLocked",
-        askPassword: "urn:scim:wso2:schema.askPassword",
-        isReadOnlyUser: "urn:scim:wso2:schema.isReadOnlyUser",
-        oneTimePassword: "urn:scim:wso2:schema.oneTimePassword",
-        profileUrl: "urn:scim:wso2:schema.profileUrl"
+        accountDisabled: "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User.accountDisabled",
```

Comment:
Lets cherry pick and send this fix separately since this is rather a unification-bug.

Reply:
https://github.com/wso2/identity-apps/pull/5471